src/ck-core/ck.C

```diff
@@ -656,6 +657,25 @@ void CkCreateChare(int cIdx, int eIdx, void *msg, CkChareID *pCid, int destPE)
   _TRACE_CREATION_DONE(1);
 }
 
+inline void CkReadyEntry(TableEntry& entry, bool nodeLevel) {
+  // the ready flag is set first to expedite the unblocking of
+  // node-level peers (that depend on the current [node]group)
+  entry.setReady();
+
+  PtrQ *ptrq = entry.getPending();
+  if(ptrq) {
+    void *pending;
+    while((pending=ptrq->deq())!=nullptr) {
+      if (nodeLevel) {
```

Comment:
Minor formatting stuff, make add spaces around operators, make spacing consistent.

Reply:
Will do, this was copy-pasted so it needs adjustment.